tests/ui/test_validation_ui.py

```diff
@@ -102,3 +102,17 @@ def test_odf_cephblockpool_compression_status(self, setup_ui_class):
             f"'Compression status' from ocs-storagecluster-cephblockpool = {compression_statuses[1]}\n"
             f"Expected: {compression_status_expected}"
         )
+
+    @ui
+    @pytest.mark.bugzilla("1994584")
+    def test_ocs_operator_is_not_present(self, setup_ui_class):
```

Comment:
The function name is bit confusing as it calls the function `verify_odf_operator_in_installed_operator()` which says verify odf operator but also validates that ocs operator is not there. So may be rename `verify_odf_operator_in_installed_operator()` to have the correct message conveyed.

Reply:
updated this!